projects/1.19/assets/no-chat-reports/nochatreports/lang/zh_cn.json

```
@@ -0,0 +1,130 @@
+	"gui.nochatreports.realms_warning.check": "不再显示",
+	"gui.nochatreports.realms_warning.learn_more": "了解更多",
+	"gui.nochatreports.signing_required.header": "警告：需要消息签名",
+	"gui.nochatreports.signing_required.contents": "该服务器要求玩家对所有聊天消息进行签名；这意味着这些消息可能被举报给Mojang，从而可能导致玩家从所有多人游戏中被禁用。很可能是服务器的所有者未关闭server.properties中的“enforce-secure-profile”选项，因为该选项从1.19.1开始就默认为启用。\n\n点击“允许签名”按钮，以对该游戏会话启用消息签名。也可以用聊天界面中的“安全状态”按钮选择对该服务器的签名策略。",
+	"gui.nochatreports.signing_required.allow_signing": "允许签名",
+	"gui.nochatreports.signing_required.cancel": "忽略",
+	"gui.nochatreports.signing_required.check": "在该游戏会话中不再显示",
+	"chat.nochatreports.disabled.signing_requested": "服务器拒绝分发未签名的消息。可以更改签名策略，或者在提示界面中操作，以启用消息签名。",
+	"gui.nochatreports.signing_mode": "§l签名策略：%1$s",
+	"gui.nochatreports.signing_mode.default": "默认值（%1$s）",
+	"gui.nochatreports.signing_mode.always": "永远签名",
+	"gui.nochatreports.signing_mode.never": "从不签名",
+	"gui.nochatreports.signing_mode.never_forced": "无法启用签名",
+	"gui.nochatreports.signing_mode.prompt": "请求玩家决策",
+	"gui.nochatreports.signing_mode.on_demand": "按需签名",
+	"gui.nochatreports.signing_mode.always.tooltip": "该模式下，在进入服务器后立刻启用消息签名。该模式与原版客户端的行为相同。",
+	"gui.nochatreports.signing_mode.never.tooltip": "该模式下，消息签名永远禁用；将会忽略服务器发送的签名请求。",
+	"gui.nochatreports.signing_mode.never_forced.tooltip": "该模式下，聊天消息无法签名。可能是玩家在使用离线账号，或者是客户端无法从Mojang服务器拉取签名密钥对。",
+	"gui.nochatreports.signing_mode.prompt.tooltip": "该模式下，聊天消息默认不签名，但如果服务器发送请求，将会显示提示界面，在此可以对该游戏会话启用消息签名。",
+	"gui.nochatreports.signing_mode.on_demand.tooltip": "与“请求玩家决策”类似，但会在服务器请求时自动启用消息签名。",
+
+	"configuration.NoChatReports.config": "No Chat Reports",
+	"configuration.NoChatReports.category.client": "客户端",
+	"option.NoChatReports.enableMod": "启用模组",
+	"option.NoChatReports.enableMod.tooltip": "启用模组在客户端的大多数功能。若在会话中更改，需要重进服务器。",
+	"option.NoChatReports.showNCRButton": "显示No Chat Report按钮",
+	"option.NoChatReports.showNCRButton.tooltip": "在多人游戏界面添加一个按钮，在此可以控制模组的大多数客户端功能。",
+	"option.NoChatReports.defaultSigningMode": "默认签名策略",
+	"option.NoChatReports.defaultSigningMode.tooltip": "设置对要求或允许消息签名的服务器所采用的默认签名策略。",
+	"option.NoChatReports.showReloadButton": "显示重连按钮",
+	"option.NoChatReports.showReloadButton.tooltip": "在多人游戏界面添加一个按钮，在此可以控制重连选项。",
+	"option.NoChatReports.verifiedIconEnabled": "启用可信服务器图标",
+	"option.NoChatReports.verifiedIconEnabled.tooltip": "允许该模组在多人游戏界面中，对有No Chat Report或其他反举报措施的服务器添加额外图标。",
+	"option.NoChatReports.verifiedIconOffsetX": "可信图标横向偏移",
+	"option.NoChatReports.verifiedIconOffsetX.tooltip": "指定“服务器安全”图标的横向偏移。",
+	"option.NoChatReports.verifiedIconOffsetY": "可信图标纵向偏移",
+	"option.NoChatReports.verifiedIconOffsetY.tooltip": "指定“服务器安全”图标的纵向偏移。",
+	"gui.NoChatReports.showEncryptionButtonWarning": "§c下述选项不会禁用聊天加密，只会隐藏配置按钮！",
+	"option.NoChatReports.showEncryptionButton": "显示加密按钮",
+	"option.NoChatReports.showEncryptionButton.tooltip": "若配置为false，仅会在聊天界面中隐藏加密按钮。注意：加密选项本身并不会随之关闭。",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
	"option.NoChatReports.showEncryptionButton.tooltip": "若配置为false，仅会在聊天屏幕中隐藏加密按钮。注意：加密选项本身并不会随之关闭。",
```
原文：**Setting this to false will simply prevent encryption button from being displayed in the chat screen. Be aware that encryption itself will not be automatically disabled along with the button.**

---

mcwiki 相关内容：

> 在多人游戏中，玩家可以通过按T与其他玩家聊天。聊天功能包括：
> 
> 历史记录 - 使用聊天框的右侧的滚动条可以查看历史聊天消息，也可上下滚动鼠标滚轮、或使用PAGEUP和PAGEDOWN。聊天框保存最近的100行消息。
> 在输入聊天文字时使用↑和↓可以调出自己的已发送消息。
> 可以点击聊天中出现的链接。
> 在聊天中使用复制（CTRL + C）和粘贴（CTRL + V）功能（部分Mac中使用⌘ Cmd代替Ctrl）。使用⇧ Shift + ↑、↓、←、→选择文本。
> 格式化字体：玩家可以使用分节符（§）更改文字样式。详见[格式化代码](https://minecraft.fandom.com/zh/wiki/%E6%A0%BC%E5%BC%8F%E5%8C%96%E4%BB%A3%E7%A0%81)。
> 注：分节符作为非法字符，本身无法在原版[客户端](https://minecraft.fandom.com/zh/wiki/%E5%AE%A2%E6%88%B7%E7%AB%AF)中输入。若将§复制粘贴到聊天框中发送，玩家会被自动踢出服务器，并提示“Illegal characters in chat（聊天消息出现非法字符）”，服务端则会报错disconnect.endOfStream。[仅[Java版](https://minecraft.fandom.com/zh/wiki/Java%E7%89%88)]
> 玩家也可以在聊天框中输入[命令](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4)。命令由服务器解析，并以斜杠（/）作为开头。
> 
> 在聊天框外，/可作为命令输入键——按下后会出现预先输入/开头的聊天框。
> 输入/后按下Tab ↹会列出当前可用的命令，与执行/[help](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4/help)命令类似。
> 在输入命令时，按下Tab ↹会根据已输入内容自动补全第一个可能的命令或用户名。若已输入内容可对应多个命令或用户名，聊天框中会显示所有可能结果的列表；再次按下Tab ↹会在输入栏内枚举这个列表。
> 
> 某些命令的一些额外参数也可能可以通过Tab ↹自动补全。
> 某些处理方块的命令（/[data](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4/data)、/[fill](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4/fill)等等）会要求提供目标方块的x、y、z轴[坐标](https://minecraft.fandom.com/zh/wiki/%E5%9D%90%E6%A0%87)。此时使用Tab ↹可将玩家当前正对方块的坐标自动填进去。
> 通过更改[聊天设置](https://minecraft.fandom.com/zh/wiki/%E9%80%89%E9%A1%B9#%E8%81%8A%E5%A4%A9%E8%AE%BE%E7%BD%AE)，聊天框的尺寸可以缩小，透明度也可以调节，甚至可以完全隐藏。

> 命令/locate
> < [命令](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4)
> [编辑](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4/locate?action=edit)
> locate
> 需要的[权限等级](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4#%E6%9D%83%E9%99%90%E7%AD%89%E7%BA%A7)	
> [Java版](https://minecraft.fandom.com/zh/wiki/Java%E7%89%88)：2
> [基岩版](https://minecraft.fandom.com/zh/wiki/%E5%9F%BA%E5%B2%A9%E7%89%88)：1
> 
> [限制条件](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4#%E9%99%90%E5%88%B6%E6%9D%A1%E4%BB%B6)	
> 启用作弊[仅[基岩版](https://minecraft.fandom.com/zh/wiki/%E5%9F%BA%E5%B2%A9%E7%89%88)]
> 在聊天框中为命令执行者显示给定类型结构、[生物群系](https://minecraft.fandom.com/zh/wiki/%E7%94%9F%E7%89%A9%E7%BE%A4%E7%B3%BB)或[兴趣点](https://minecraft.fandom.com/zh/wiki/%E5%85%B4%E8%B6%A3%E7%82%B9%E6%96%87%E4%BB%B6%E6%A0%BC%E5%BC%8F)[仅[Java版](https://minecraft.fandom.com/zh/wiki/Java%E7%89%88)]的最近坐标和距离。在[Java版](https://minecraft.fandom.com/zh/wiki/Java%E7%89%88)中，返回的坐标可以点击以自动在聊天框中输入/[tp](https://minecraft.fandom.com/zh/wiki/%E5%91%BD%E4%BB%A4/tp) `@s `坐标。

那看起来应该是“聊天框”

---

Here

https://github.com/CFPAOrg/Glossary/blob/104816b3d8ff6520dccdba9ab5a8a51b68c31546/Minecraft%20Vanilla%201.18.2.csv#L2013

---

confirmed
那估计聊天框是下面那个文本框.......